activerecord/CHANGELOG.md

```diff
@@ -469,6 +469,18 @@
     end
     ```
 
+    Active Record logs will also include timing info for the duration of how long
+    the async query waited for the mutex lock. This timing is useful to know how
+    effecient or not the query is.
```

Comment:
> how long the async query waited for the mutex lock

I don't think this is very easy to understand. I'm not a native speaker, so I'm not very good at this, but maybe something like:

> how long the main thread had to wait to access the results.

?

Also:

> This timing is useful to know how effecient or not the query is.

Might be clearer as:

> wether it's worth loading asynchronously.

?

Reply:
@byroot Thanks again for your review! I've updated with your suggestions in af8a37ee48 if you can take another look :bow: